src/app/plugins/plugin_colorcalib.cpp

```diff
@@ -140,3 +153,67 @@ void PluginColorCalibration::mouseReleaseEvent ( QMouseEvent * event, pixelloc l
+    if (v_lut_sources->getCount()==1 && v_lut_sources->getString()==LUT_COPY_PLACEHOLDER) {
+        VarString *pSelf = reinterpret_cast<VarString*>(settings->getParent());
+        std::vector<VarType *> vectRelatives = settings->findRelatives("LUT 3D", true);  //find LUT3D nodes node
+        if (vectRelatives.size() < 1) {     //nothing like LUTs? abort
+            fprintf(stderr, "LUT3D: Could not find any LUT3D nodes for sources, aborting copy.\n");
+        }
+        else {
+            v_lut_sources->setSize(0);          //truncate/clear former list
+            for (int iCam=0; iCam<vectRelatives.size(); iCam++) {
+                VarString *pCamera = reinterpret_cast<VarString*>(vectRelatives[iCam]->getParent()->getParent());
+                if (pCamera != pSelf)
+                    v_lut_sources->addItem(pCamera->getName());
+            }
+            if (!vectRelatives.empty())
+                v_lut_sources->selectIndex(0);
+            v_lut_colors->setSize(0);          //truncate/clear former list
+            v_lut_colors->addItem("all");
+            v_lut_colors->selectIndex(0);
+        }
+    }
+    else {              //user has selected a camera
+        bool bSuccess = true;
+        VarBlob *pSource = NULL;
+        int color_index = -1;       //default -1 is ALL colors; TODO be from var 'v_lut_colors'
+        std::vector<VarType *> vectTarget = settings->findRelatives(v_lut_sources->getString(), true);
+        if (vectTarget.size()!=1) {
+            fprintf(stderr, "LUT3D: Found too many camera nodes (%d) for camera '%s', aborting copy.\n",
+                    static_cast<int>(vectTarget.size()), v_lut_sources->getString().c_str());
+            bSuccess = false;
+        }
+        else {
+            vectTarget = vectTarget[0]->findRelatives("LUT Data", false);  //just children
+        }
+        if (bSuccess && vectTarget.size()!=1) {
+            fprintf(stderr, "LUT3D: Found too many LUT nodes (%d) for camera '%s', aborting copy.\n",
+                    static_cast<int>(vectTarget.size()), v_lut_sources->getString().c_str());
+            bSuccess = false;
+        }
+        else {
+            pSource = reinterpret_cast<VarBlob*>(vectTarget[0]);
+            
+            //WARNING: Makes assumption that all LUT blobs have the same properties
+            //  (e.g. same initializer, etc) because we do not reallocate, modify shift
+            //  indicies, or anything else dramatic.  This is an acceptable risk because
+            //  the constructor for this class just loops to add multiple camera sources.
+            //  (added corresponding warning in multistack_robocup_ssl.cpp at allocation)
+            
+            GLLUTWidget *pGLLUTw = lutw->getGLLUTWidget();
+            if (pGLLUTw->copyLUT(pSource->getDataPointer(), pSource->getDataSize(), color_index)) {
+                fprintf(stderr, "PluginColorCalibration: Successful copy of LUT data from camera '%s'.\n",
```

Comment:
Would be useful to include in the print statement what the destination camera ID is.